toothpick-compiler/src/main/java/toothpick/compiler/common/ToothpickProcessor.java

```diff
@@ -591,4 +593,8 @@ private TypeElement getKindParameter(Element element) {
     TypeMirror firstParameterTypeMirror = ((DeclaredType) elementTypeMirror).getTypeArguments().get(0);
     return (TypeElement) typeUtils.asElement(typeUtils.erasure(firstParameterTypeMirror));
   }
+
+  protected CodeGenerator createRegistryGenerator(RegistryInjectionTarget target) {
+    return new RegistryGenerator(target, typeUtils);
```

Comment:
should we inline the method now?

Reply:
If you want. I let it stay because it's called from 2 places, a very minor code duplication, but don't really care either way.

Reply:
yeah, but in the end it is just one line. So we are adding more lines than the ones we are removing/saving.

Reply:
it's not only about line count :) if we decide to change the body we can do it only in one place

Reply:
of course it is not the number of lines. But we are saving a line where we are just "newing" a class. If we changed the constructor, we would need to change the method probably as well. And moreover, we are not saving lines just making the code navigation more complex. 

So i don't see the benefit, as we are duplicating the method call anyway.

Reply:
Inlined